packages/react-components/src/components/styles/SendBox.styles.ts

```diff
@@ -20,6 +20,26 @@ export const sendBoxStyleSet = {
   }
 };
 
+/**
+ * @private
+ */
+export const sendBoxStyleMain = mergeStyles({
+  padding: '0rem',
+  display: 'flex',
+  flexDirection: 'column',
+  alignItems: 'flexStart',
+  border: '1px solid',
+  maxHeight: '200px',
+  overflowX: 'hidden',
+  overflowY: 'auto',
+  borderRadius: '0.25rem',
+  borderColor: 'grey',
+  borderStyle: 'solid',
+  ':hover': { borderColor: '#0078D4' },
```

Comment:
For colors, please use the theme palette.
`theme.palette.{colorname}`

Reply:
done